docs/Reference/web3js-eea-Methods.md

```diff
@@ -14,7 +14,7 @@ The Options parameter has the following properties:
 * `privateKey`: Ethereum private key with which to sign the transaction.
 * `privateFrom` : Orion public key of the sender.
 * [`privateFor` : Orion public keys of recipients or `privacyGroupId`: Privacy group to receive the transaction](../Concepts/Privacy/Privacy-Groups.md)
-* `nonce` : Optional. If not provided, calculated using [`eea_getTransctionCount`](API-Methods.md).
+* `nonce` : Optional. If not provided, calculated using [`priv_getEeaTransactionCount`](API-Methods.md#priv_geteeatransactioncount).
```

Comment:
Won't this be calculated by `priv_getEeaTransactionCount` if `privateFor` is provided and by `priv_getTransactionCount` if `privacyGroupId` is provided?

Reply:
done